### PR TITLE
Docs: Fix docker-compose instructions

### PR DESCRIPTION
**Pull request by ofhope**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

The main concern here was the command `docker-compose run web rails data_updates:run`. There is no named instance `web` in docker compose. Changing it to `rails rails` reads poorly but executes. Also a side note if someone already has a running instance `docker-compose exec rails rails data_updates:run` is faster as it doen't have to boot a new instance.

I was confused with the language use **stand up** I'm pressuming it's meant to be **start up** but wasn't sure if this is some community language I'm unfamiliar with.
> from standing up new communities.

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] Yes
- [x] No, and this is why: Documentation only PR
- [ ] I need help with writing tests

## Added to documentation?

- [x] Docs.forem.com
- [ ] README
- [ ] No documentation needed

### CODE REVIEW

File: docs/getting-started/pull-request.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Preparing a Pull Request
 
 - Try to keep the pull requests small. A pull request should try its very best
   to address only a single concern.
-- If you plan to do further work after the PR is submit, please use the
+- If you plan to do further work after the PR has been submited, please use the
```
Comment (rhymes) [inline docs/getting-started/pull-request.md:9]: a `t` in submitted is missing here

File: docs/self-hosting.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ platform.
 
 We aren't quite ready for mainstream adoption of our platform,
 [Forem](https://forem.com), but that hasn't stopped some adventurous developers
-from standing up new communities.
+from starting up new communities.
```
Comment (rhymes) [inline docs/self-hosting.md:14]: this was actually correct, "to stand up" means "to erect"

**Review by rhymes — CHANGES_REQUESTED** [review]
Thanks @ofhope, there's a typo and a change that needs to be reverted but great work for catching the misalignment between the docker compose file and the documentation

File: docs/self-hosting.md (hunk 2 above)
Comment (ofhope) [reply docs/self-hosting.md:14]: Yeah OK, I was wondering if I was just unfamiliar with it! Thanks for clarifying.

**Review by jdoss — APPROVED** [review]

**Review by rhymes — APPROVED** [review]
Thanks!

**Pull request merged**